app/bundles/EmailBundle/MonitoredEmail/Processor/Bounce/BouncedEmail.php

```diff
@@ -114,10 +111,7 @@ public function setType($bounceType)
         return $this;
     }
 
-    /**
-     * @return int
-     */
-    public function isFinal()
+    public function isFinal(): bool
```

Comment:
Is changed by the rector.

Reply:
I got the same problem on my PR. I solved it like this: https://github.com/mautic/mautic/pull/11251/commits/ef9a1896698c828f58d11d5422d01ca3957659c9

For M6 we'll remove that exclusion and let it do its work. I still don't want to disable that rule completely as it will help with all new methods having return types.

Reply:
@escopecz I'd say this is a very bad idea. Because every new commit with new methods will not be checked and this will some devs will write already legacy code.

Reply:
That's why I didn't disable the rule for the whole codebase but only for the 2 files that needed it.

@mollux what's your take on this? It's lesser evil to break the BC by allowing Rector to add the type or disabling the rule on those files to avoid the BC for the lifespan of M5?

Reply:
@escopecz this is the only change that's left to change or keep. WDYT?

Reply:
The chance that some plugin is extending such service is pretty small. I'd say let's close an eye when Rector is forcing us to add these obvious types.